extender/types.go

```diff
@@ -17,6 +19,18 @@ type Server struct {
 	Scheduler
 }
 
+type NotBindingExtender struct{}
+
+func (m *NotBindingExtender) Bind(w http.ResponseWriter, r *http.Request) {}
```

Comment:
This method of negative interfaces is interesting, but it adds complexity to the package here. The simplest way to do this would be to add a Bind() method to TAS that just returns a 404. This is a good deal more complex to understand I think, but it does create a pattern for not meeting the interface which I like quite a lot. What do you think are the tradeoffs here?

If we're going to do this in this way these functions should return 404s or some other understandable http code instead of being blank. Right now the behavior in TAS is a 404 at an unknown endpoint, so it might be good to keep that implemented.

Reply:
I'll simplify by removing